docs/fiddles/features/offscreen-rendering/main.js

```diff
@@ -0,0 +1,28 @@
+const { app, BrowserWindow } = require('electron')
+const fs = require('fs')
+
+app.disableHardwareAcceleration()
+
+let win
+
+app.whenReady().then(() => {
+  win = new BrowserWindow({ webPreferences: { offscreen: true } })
+
+  win.loadURL('http://github.com')
+  win.webContents.on('paint', (event, dirty, image) => {
+    fs.writeFileSync('ex.png', image.toPNG())
```

Comment:
Maybe add a line that logs the fact that `fs.writeFileSync` is adding the image to `${process.cwd()}/ex.png` so that people can see that the demo is working